hedera-node/src/main/java/com/hedera/services/store/tokens/unique/ExceptionalUniqueTokenStore.java

```diff
@@ -0,0 +1,208 @@
+/*
+ *
+ *  * ‌
+ *  * Hedera Services Node
+ *  * ​
+ *  * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ *  * ​
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * y‌
+ * Hedera Services Node
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.ou may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *  * ‍
+ *
+ *
+ */
+
+package com.hedera.services.store.tokens.unique;
+
+import com.hedera.services.ledger.HederaLedger;
+import com.hedera.services.ledger.TransactionalLedger;
+import com.hedera.services.ledger.properties.AccountProperty;
+import com.hedera.services.state.merkle.MerkleAccount;
+import com.hedera.services.state.merkle.MerkleToken;
+import com.hedera.services.state.submerkle.RichInstant;
+import com.hedera.services.store.CreationResult;
+import com.hedera.services.store.tokens.TokenStore;
+import com.hederahashgraph.api.proto.java.AccountID;
+import com.hederahashgraph.api.proto.java.ResponseCodeEnum;
+import com.hederahashgraph.api.proto.java.TokenCreateTransactionBody;
+import com.hederahashgraph.api.proto.java.TokenID;
+import com.hederahashgraph.api.proto.java.TokenMintTransactionBody;
+import com.hederahashgraph.api.proto.java.TokenUpdateTransactionBody;
+
+import java.util.List;
+import java.util.function.Consumer;
+
+public enum ExceptionalUniqueTokenStore implements TokenStore, UniqueStore {
```

Comment:
Much of this code can be removed based on the comment for default interface methods

Reply:
Those will be removed after Michael's refactoring